src/main.ts

```diff
@@ -30,14 +30,19 @@ export const cspClient = clients.newClient(
   }
 )
 
+const defaultHeadersVibClient = {
+  "Content-Type": "application/json",
+  "User-Agent": `vib-action/${userAgentVersion}`,
+}
+
 export const vibClient = clients.newClient(
   {
     baseURL: `${process.env.VIB_PUBLIC_URL ? process.env.VIB_PUBLIC_URL : constants.DEFAULT_VIB_PUBLIC_URL}`,
     timeout: getNumberInput("http-timeout"),
-    headers: {
-      "Content-Type": "application/json",
-      "User-Agent": `vib-action/${userAgentVersion}`,
-    },
+    headers:
+      process.env.VERIFICATION_MODE === "true"
+        ? { ...defaultHeadersVibClient, "x-verification-mode": "SERIAL" }
```

Comment:
Ok, this was a bit misleading. I thought (see comment above) the env var would define the verification mode. Not a boolean. If this is a boolean then the name of the variable should change to something like `serial-verification`. If not a boolean, then this line should use the value of the variable. 

I think my preference would be to let users specify the verification mode. There is nothing preventing us from adding new verification modes in the future, so the boolean approach is self-limiting.